### PR TITLE
False positives

### PR DESCRIPTION
**Pull request by blakeblackshear**
- False positive submissions for Frigate+
- Also includes the detection as an annotation for true positives

image

### CODE REVIEW

File: frigate/events.py [hunk 1]
```diff
@@ -85,81 +82,82 @@ def run(self) -> None:
...
-                    box=event_data["box"],
-                    area=event_data["area"],
+                    region=region,
+                    box=box,
                     has_clip=event_data["has_clip"],
                     has_snapshot=event_data["has_snapshot"],
+                    model_hash=first_detector.model.model_hash,
+                    model_type=first_detector.model.model_type,
+                    detector_type=first_detector.type,
                 ).execute()
```
Comment (NickM-27) [inline frigate/events.py:155]: I think this means that in-progress events that have `sub_label` or `retain_indefinitely` will have their values erased when this is called

Comment (NickM-27) [reply frigate/events.py:155]: maybe there is some way to keep track if the event has been inserted or not, and call replace or update based on that?

Comment (NickM-27) [reply frigate/events.py:155]: https://stackoverflow.com/a/33819556 seems to have some good options

**Comment by ehn** [comment]
In case something is a false positive but also is something that should have been detected, say a deer or a person detected as a dog, will there be a way to include that in the submission? Or should we just upload and label as we do today in such cases?

**Comment by blakeblackshear** [comment]
You will have to add the additional labels in Frigate+.

This also replaces the "Sent to Frigate+" text with an "Edit in Frigate+" link.

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**